file-updates/test.sh

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env sh
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+DEV_KUBECONFIG="--kubeconfig=$SCRIPTDIR/../kubeconfig/stage_dev_release_kubelogin"
+MANAGED_KUBECONFIG="--kubeconfig=$SCRIPTDIR/../kubeconfig/stage_managed_release_kubelogin"
+
+MANAGED_NAMESPACE="managed-release-team-tenant"
+APPLICATION_NAME="fileupdatestest"
+COMPONENT_NAME="fileupdatestest-component"
+RELEASE_PLAN_NAME="file-updates-test-rp"
+RELEASE_PLAN_ADMISSION_NAME="file-updates-test-rpa"
+RELEASE_STRATEGY_NAME="file-updates-test-push-rs"
+TIMEOUT_SECONDS=600
+
```

Comment:
I am wondering if we should set these values as an external _kind of_ config file. It is the second time I see this same script (we have the same for FBC), so I guess we should make somehow a reusable script and then just set the variables outside. Not required for this one, just an idea.

Reply:
these scripts will eventually move to the e2e framework. See https://issues.redhat.com/browse/HACBS-2597

FYI @kasemAlem